Not sure why the build is failing on Android from `th72d.c` when that file wasn't even touched:

``` console
  CC       thd72.lo
thd72.c:1469:11: error: call to undeclared function 'write_block'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
 1469 |     ret = write_block(rp, cmd, CMD_SZ);
      |           ^
thd72.c:1477:11: error: call to undeclared function 'read_block'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
 1477 |     ret = read_block(rp, resp, CMD_SZ);
      |           ^
2 errors generated.
make[1]: *** [Makefile:576: thd72.lo] Error 1
make[1]: *** Waiting for unfinished jobs....
thd74.c:1461:11: error: call to undeclared function 'write_block'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
 1461 |     ret = write_block(rp, cmd, CMD_SZ);
      |           ^
thd74.c:1469:11: error: call to undeclared function 'read_block'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
 1469 |     ret = read_block(rp, resp, CMD_SZ);
      |           ^
2 errors generated.
make[1]: *** [Makefile:576: thd74.lo] Error 1
make: *** [Makefile:666: all-recursive] Error 1
Error: Process completed with exit code 2.
```

--- a/rigs/kenwood/thd72.c
+++ b/rigs/kenwood/thd72.c
@@ -29,7 +29,6 @@
 #include "kenwood.h"
 #include "th.h"
 #include "misc.h"
-#include "iofunc.h"
 
 // Some commands are very slow to process so we put a DELAY in those places
 #define DELAY hl_usleep(300*1000)
@@ -1450,7 +1449,7 @@ static int thd72_get_channel(RIG *rig, vfo_t vfo, channel_t *chan,
     return RIG_OK;
 }
 
-#ifdef false  /* not working */
+#if 0  /* not working */
 #define CMD_SZ 5
 #define BLOCK_SZ 256
 #define BLOCK_COUNT 256

--- a/rigs/kenwood/thd74.c
+++ b/rigs/kenwood/thd74.c
@@ -29,7 +29,6 @@
 #include "kenwood.h"
 #include "th.h"
 #include "misc.h"
-#include "iofunc.h"
 
 #define THD74_MODES (RIG_MODE_FM|RIG_MODE_AM|RIG_MODE_LSB|RIG_MODE_USB|RIG_MODE_CW|RIG_MODE_FMN|RIG_MODE_WFM|RIG_MODE_CWR)
 #define THD74_MODES_TX  (RIG_MODE_FM)
@@ -1442,7 +1441,7 @@ int thd74_set_split_freq(RIG *rig, vfo_t vfo, freq_t tx_freq)
     return -RIG_EPROTO;
 }
 
-#ifdef false    /* not working */
+#if 0    /* not working */
 #define CMD_SZ 5
 #define BLOCK_SZ 256
 #define BLOCK_COUNT 256

--- a/src/rig.c
+++ b/src/rig.c
@@ -8837,6 +8837,7 @@ static void *morse_data_handler(void *arg)
     free(c);
 
     pthread_exit(NULL);
+    return NULL;             // May get a warning if pthread_exit() has [[noreturn]]
 }
 
 

